core/templates/dev/head/filters.js

```diff
@@ -342,6 +342,15 @@ oppia.filter('summarizeNonnegativeNumber', [function() {
   };
 }]);
 
+oppia.filter('removeExtraLines', [function() {
```

Comment:
I would add a comment to this function explaining more specifically what this is for (stripping empty lines in HTML content) and when an empty line would be stripped.